internal/charmstore/search_test.go

```diff
@@ -77,6 +76,8 @@ func (s *StoreSearchSuite) TestSuccessfulExport(c *gc.C) {
 		readACLs := []string{params.Everyone}
 		if r.User != "" {
```

Comment:
Hmm, I think I'd prefer it to be more direct and hold a slice of structs each holding a URL and whether it's promulgated.